src/helpers.js

```diff
@@ -668,13 +668,10 @@ module.exports = Object.assign({}, utils, packages, {
 
   getJavaInfo: () => {
     utils.log('trace', 'getJavaInfo');
-    if (macos || linux) {
-      return Promise.all([
-        utils.run('javac -version 2>&1').then(v => utils.findVersion(v, /\d+\.[\w+|.|_|-]+/)),
```

Comment:
2>&1 should work on windows from my research?

Reply:
It works on cmd, not on powershell.

Reply:
Do you want to change it ? I don't know whether there is a potential compatibility problem.